.changeset/tidy-emus-hammer.md

```diff
@@ -0,0 +1,7 @@
+---
+'@hashicorp/react-actions': major
+'@hashicorp/react-intro': major
+'@hashicorp/react-next-steps': major
+---
```

Comment:
@alexcarpenter - just want to triple check that we want all packages to be major updates since this introduces a breaking change, right?

Reply:
I think am thinking so. Does that sound right @dstaley?

Reply:
If I'm understanding correctly, only `react-actions` should have a changeset entry. You'd remove the changes to the rests for `intro` and `next-steps`, and update them to the new major version in a subsequent PR. Also, I believe you might need to merge in changes from `main` as it looks like some of your `package-lock.json` changes are from other PRs?

Reply:
Ahhh, that makes sense. ⚡  So, this is what I need to do:

- Update changeset such that there's only a major bump for `actions`
- Revert changes to `intro` and `next-steps`
- Merge in / publish
- Create separate PRs for `intro` and `next-steps` changes that include bumping to the new `actions` version ☝️

Reply:
Yup! That's the general procedure for releasing a change that has to be fixed/upgraded to in dependent packages.